scripts/skills/perks/perk_rf_angler.nut

```diff
@@ -15,12 +15,19 @@ this.perk_rf_angler <- ::inherit("scripts/skills/skill", {
 
 	function onAnySkillExecuted( _skill, _targetTile, _targetEntity, _forFree )
 	{
-		if (_skill.getID() == "actives.throw_net" && !_forFree)
+		if (_skill.getID() == "actives.throw_net")
 		{
-			_targetEntity.getSkills().getSkillByID("actives.break_free").m.ChanceBonus -= 20;
+			local breakFreeSkill = _targetEntity.getSkills().getSkillByID("actives.break_free");
+			breakFreeSkill.setBaseValue("ActionPointCost", ::Math.floor(breakFreeSkill.getBaseValue("ActionPointCost") * 1.5));
```

Comment:
`1.5` should be a new `this.m.BreakFreeCostMult` instead so that this value is much easier to read out or change by submods.